examples/Boost-python/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2013, Ruslan Baratov
+# Modified work: Copyright (c) 2018, Gregory Kramida
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+option(HUNTER_BUILD_SHARED_LIBS "..." ON)
+
+# Configure:
+set(REQUESTED_PYTHON_VERSION 3.5)
```

Comment:
We need 3.5 for Travis: https://github.com/ruslo/hunter/pull/1668/commits/139d1102ffdbf4a5c97139948424902ab70d9e47

Reply:
Perhaps change or remove comment on line 18 to match this.